### PR TITLE
chore: rename, rekres

### PR DESCRIPTION
**Pull request by smira**
Rename Go module, rekres, update.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,11 +1,11 @@
-module github.com/talos-systems/net
+module github.com/siderolabs/net
 
 go 1.17
```
Comment (frezbo) [inline go.mod]: 1.19?

**Review by frezbo — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (smira) [reply go.mod]: oops, yes, should be 1.19

**Pull request merged**